src/PopUpTodo.js

```diff
@@ -1,19 +1,42 @@
-import React from "react";
+import React,{useState} from "react";
 import "./PopUpTodo.css";
 
-const PopUpTodo = () => {
+const PopUpTodo = ({addTodo,HandleChange,inputTodo}) => {
+  // const [todos, settodos] = useState("");
+  // const [list, setlist] = useState([]);
+
+
+
+  // const inputTodo=(e)=>{
+  //   settodos(e.target.value)
+    
+  // }
+
+
+  // const HandleChange=(e)=>{
+  //   e.preventDefault()
+  //   const newtodo={ id:Math.random(), name:name, check: false }
+  //   addTodo(newtodo)
+  //   setname("")
+
+  // }
+
+
+  // const addToList=(e)=>{
+  //   setlist(e.target.value)
+  // }
   return (
     <>
       <div className="input_card">
-        <div>
-          <h2>Add Todo</h2>
-
-          <textarea className="input_area"></textarea>
-        </div>
-        <div className="popup_buttons_div">
-          <button>Cancel</button>
-          <button>Done</button>
-        </div>
+        <h2>Add Todo</h2>
+        <form onSubmit={HandleChange}>
+          <input className="input_area" required resize="none" onChange={inputTodo}></input>
```

Comment:
inputTodo why do you need it here?
you can create a state variable and use it in input filed, when submitting the form pass data to addTodo function @Kunal-inno